app.py

```diff
@@ -521,12 +521,14 @@ def get_quiz(level_source, question_nr, attempt):
     question = quiz.get_question(quiz_data, question_nr)
     question_obj = quiz.question_options_for(question)
 
+    chosen_letter = 'X' if session.get('chosenOption') is None else session.get('chosenOption')
     return render_template('quiz_question.html',
                            quiz=quiz_data,
                            level_source=level_source,
                            questionStatus=questionStatus,
                            questions=quiz_data['questions'],
                            question_options=question_obj,
+                           chosen_letter = chosen_letter,
```

Comment:
Misschien moeten we dit, met het ook op de toekomst, geen chosen_letter maar chosen_option noemen dat dit over een tijdje geen letters meer zijn?

Reply:
Goed punt! Ik heb het veranderd.